src/error_page.rs

```diff
@@ -8,7 +8,7 @@ pub(crate) fn map_error(
   result.unwrap_or_else(|error| {
     error.print_backtrace(&mut stderr);
     writeln!(stderr, "{}", error).ok();
-    let mut response = html::wrap_body(html! {
+    let mut response = html::wrap_body("Agora - Error".to_string(), html! {
```

Comment:
I like the idea of including the error text, so this could also be:

```suggestion
    let mut response = html::wrap_body(&error.status().to_string(), html! {
```